zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -558,14 +558,14 @@ paths:
       responses:
         "200":
           description: >
-            The User Task Search successful response.
+            The user task search successful response.
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/UserTaskSearchQueryResponse"
         "400":
           description: >
-            The User Task Search Query failed.
+            The user task search Query failed.
```

Comment:
❓ Should `Query` also be lowercase here? I can see the other query endpoints also use uppercase but I was wondering why that is. In @christinaausley's [original commit](https://github.com/camunda/camunda-docs/pull/4178/commits/45b9e61a03d355cde8de48139f1f1e273db7000a), it was all lowercase.

Reply:
Yes, this is a mistake on my part! Fixing.